src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateEndConstruct/GenerateEndConstructTests.vb

```diff
@@ -45,7 +45,7 @@ End Using</MethodBody>
             Dim text = <File>
 Structure Goo[||]</File>
 
-            Dim expected = StringFromLines("Structure Goo", "End Structure", "")
+            Dim expected = StringFromLines("", "Structure Goo", "End Structure", "")
```

Comment:
😕 Why does this one not produce a blank line in the output line the others below do? I'll file a bug, but not sure if it's something to change.